packages/framework/src/Model/Complaint/Status/Grid/ComplaintStatusInlineEdit.php

```diff
@@ -34,6 +38,12 @@ public function __construct(
      */
     protected function createEntityAndGetId($complaintStatusData): int
     {
+        if (!$this->domain->hasAdminAllDomainsEnabled()) {
+            throw new InvalidFormDataException([
+                t('Creating a record requires all domains to be enabled as domain-specific fields cannot be empty.'),
```

Comment:
maybe there could be a hint, about what to do - something like, "if you want to proceed, select all domains in Domain filter in the header first"

Reply:
added, good idea